Patch-Source: https://gitlab.archlinux.org/archlinux/packaging/packages/python-httplib2/-/blob/3292e2dd80c986ae6f8bd37bab1db8065e3c3318/use-system-certificates.patch
--- a/python2/httplib2/certs.py
+++ b/python2/httplib2/certs.py
@@ -19,9 +19,7 @@ except ImportError:
     pass
 
 
-BUILTIN_CA_CERTS = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "cacerts.txt"
-)
+BUILTIN_CA_CERTS = "/etc/ssl/certs/ca-certificates.crt"
 
 
 def where():
--- a/python3/httplib2/certs.py
+++ b/python3/httplib2/certs.py
@@ -19,9 +19,7 @@ except ImportError:
     pass
 
 
-BUILTIN_CA_CERTS = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "cacerts.txt"
-)
+BUILTIN_CA_CERTS = "/etc/ssl/certs/ca-certificates.crt"
 
 
 def where():
--- a/tests/test_cacerts_from_env.py
+++ b/tests/test_cacerts_from_env.py
@@ -9,7 +9,7 @@ import tempfile
 import httplib2
 
 
-CA_CERTS_BUILTIN = os.path.join(os.path.dirname(httplib2.__file__), "cacerts.txt")
+CA_CERTS_BUILTIN = "/etc/ssl/certs/ca-certificates.crt"
 CERTIFI_CERTS_FILE = "unittest_certifi_file"
 CUSTOM_CA_CERTS = "unittest_custom_ca_certs"
 
